src/language-css/print/parenthesized-value-group.js

```diff
@@ -93,8 +98,12 @@ function printParenthesizedValueGroup(path, options, print) {
       child.groups[0].type !== "value-paren_group" &&
       child.groups[2]?.type === "value-paren_group"
     ) {
-      const { parts } = doc.contents.contents;
-      parts[1] = group(parts[1]);
+      doc = mapDoc(doc, (d) => {
+        if (getDocType(d) === DOC_TYPE_LABEL && d.label === "colon") {
+          return group(d);
+        }
+        return d;
+      });
```

Comment:
It's already grouped, do we still need this?

(I remember I asked this yesterday, but maybe forgot to hit button? Can't find it)